src/App.js

```diff
@@ -1,16 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  // console.log(chatMessages);
+
+  // const [likedMap, setLikedMap] = useState({})
+  const [likedCount, setLikedCount] = useState(0);
+
+  const addLike = () => setLikedCount(likedCount + 1);
+  const removeLike = () => setLikedCount(likedCount - 1);
```

Comment:
Can you think of how you could combine the logic from addLike() and removeLike() into one method like updateLike()? The bodies of these 2 methods look very similar, the difference being + or - so maybe we can add some logic by passing in a boolean value 
```javascript
updateLike = isLiked => { // your logic in here }

Reply:
Another comment about updating likedCount state with setLikedCount. When state represents something like a counter, we need to depend on a state's previous value and increment the previous value.

`setLikedCount(prevState => prevState + 1)`

This video [here](https://www.youtube.com/watch?v=yvTGXH7uybA) does a great job explaining why